generate_influxdb_iox_protobuf.sh

```diff
@@ -2,6 +2,11 @@
 
 set -e
 
+# If either of the commands fail below, run the following and then retry:
+#
+# go get google.golang.org/protobuf/cmd/protoc-gen-go@v1.26
+# go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@1.39.0
+
```

Comment:
This is supposed to be fixed by lines 5~12 below. Did that not work?

Reply:
I got an error on my machine. 

This is what happened when I ran `generate_influxdb_iox_protobuf.sh` prior to running those commands:

```shell
alamb@MacBook-Pro influxdb-iox-client-go % ./generate_influxdb_iox_protobuf.sh 
./generate_influxdb_iox_protobuf.sh 
installing protoc-gen-go-grpc
go install: google.golang.org/grpc/cmd/protoc-gen-go-grpc@1.39.0: google.golang.org/grpc/cmd/protoc-gen-go-grpc@1.39.0: invalid version: unknown revision 1.39.0
```